server/services/caseloadService.test.ts

```diff
@@ -1678,66 +1678,4 @@ describe('Caseload Service', () => {
       expect(CaseloadService.isEligibleEDS(yesterday, tenDaysFromNow, tenDaysFromNow, nineDaysFromNow)).toBe(false)
     })
   })
-
-  describe('getHardStopReferenceDate', () => {
```

Comment:
Is this not still in use for `NOT_STARTED` licences? If so, I think we should keep the specs for it